package-lock.json

```diff
@@ -1,8 +1,15953 @@
 {
   "name": "hmpps-interventions-ui",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Do you think you could remove this change (the whole `package-lock`) from the commit? It probably did this by itself when we ran `npm install` earlier. What do you get when you run `npm -v` in the project?

I think this is something we should make standard on the project, or find a workaround for, as I just tried to use the latest verson and it updated `package.lock`. Probably best to fix that in another PR though.